lib/babel.js

```diff
@@ -35,13 +62,26 @@ export function createConfiguration ({
 	}
 }
 
+/**
+ * Run babel transform, return a promise that would resolve with the code
+ * @param {string} code the source code
+ * @param {ConfigurationOptions} options options to pass to createConfiguration
+ * @returns {Promise<string>} the compiled code
+ */
 let transform = (code, options) =>
 	new Promise((resolve, reject) => {
 		babel.transform(code, createConfiguration(options), (error, result) => {
 			error ? reject(error) : resolve(result.code)
 		})
 	})
 
+/**
+ * Build a given filepath to a target path with our babel config
+ * @param {string} sourceFile the path to the source code
+ * @param {string} targetFile the path to output on
+ * @param {ConfigurationOptions} options Options to pass to createConfiguration
+ * @returns {Promise.<void>} resolves when done, rejects if couldn't
```

Comment:
Should this be:

```suggestion
 * @returns {Promise<void>} resolves when done, rejects if couldn't
```

(basing this assumption on the previous method using `{Promise<string>}`)

Reply:
oh, i think it's meant to be `Promise.<string>` in the other place